deploy/configmap.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
```

Comment:
Why this cm was added ?

Reply:
https://github.com/redhat-developer/kubernetes-image-puller/commit/fe93ea45eb93514436d5aaeaff48b00844483631#diff-04fe262db52ef4374244bc675f347ebf on 4 Apr 2019

Reply:
ok, but why do you want to copy-paste the downstream cm to upstream single-cluster puller. I feel like you are missing the point that those two are different beasts

Reply:
ok, then when you have an upstream example to provide, I can use that instead.

The point was to have something referenced in the dockerfile so that downstream knows how to use this image.

Reply:
The nightly tags are intentional.  The 7.0.0 not so much.  I will change that

Reply:
you may discover the plugin runner has moved / been ernamed. 

```
docker pull eclipse/che-remote-plugin-runner-java8:7.8.0
  manifest unknown: manifest unknown
```

Reply:
Thanks for letting me know.  I just copied from the redhat-developer repo and left whatever was in there